mmhuman3d/models/body_models/smplx.py

```diff
@@ -371,5 +392,330 @@ def forward(self,
             output['vertices'] = smplx_output.vertices
         if return_full_pose:
             output['full_pose'] = smplx_output.full_pose
+        return output
+
 
+@dataclass
+class ModelOutput(SMPLXOutput):
+    """The name of the model output."""
+    smpl_joints: Optional[torch.Tensor] = None
+    joints_J19: Optional[torch.Tensor] = None
+    smplx_vertices: Optional[torch.Tensor] = None
+    flame_vertices: Optional[torch.Tensor] = None
+    lhand_vertices: Optional[torch.Tensor] = None
+    rhand_vertices: Optional[torch.Tensor] = None
+    lhand_joints: Optional[torch.Tensor] = None
+    rhand_joints: Optional[torch.Tensor] = None
+    face_joints: Optional[torch.Tensor] = None
+    lfoot_joints: Optional[torch.Tensor] = None
+    rfoot_joints: Optional[torch.Tensor] = None
+
+
+class SMPLX_ALL(nn.Module):
```

Comment:
To maintain clean body model classes, we may take some efforts to review the difference between `SMPLX_ALL` and `SMPLX`, to reuse the similar functions as much as possible.

Reply:
`SMPLX_ALL` is an extension of the `SMPLX`, with support for gender selection in `forward`. I move `SMPL_ALL` to `mmhuman3d/models/heads/pymafx_head.py`.

Reply:
Renaming it to `GenderedSMPLX` will be fine.